#!/usr/bin/env python
PACKAGE = "lidar_calibration"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("range1", double_t, 0, "the range of LiDAR", 8, 0.0, 30.0)
gen.add("height1", double_t, 0, "the height of LiDAR", 1.09, -2.0, 2.0)
gen.add("pitch1", double_t, 0, "the pitch angle of LiDAR", 0, -90.0, 90.0)
gen.add("roll1", double_t, 0, "the roll angle of LiDAR", 0, -5.0, 5.0)
gen.add("yaw1", double_t, 0, "the yaw angle of LiDAR", 0, -90.0, 90.0)

gen.add("range2", double_t, 0, "the range of LiDAR", 8, 0.0, 30.0)
gen.add("height2", double_t, 0, "the height of LiDAR", 1.0, -2.0, 2.0)
gen.add("pitch2", double_t, 0, "the pitch angle of LiDAR", 0, -90.0, 90.0)
gen.add("roll2", double_t, 0, "the roll angle of LiDAR", 0, -5.0, 5.0)
gen.add("yaw2", double_t, 0, "the yaw angle of LiDAR", 0, -90.0, 90.0)

exit(gen.generate(PACKAGE, "lidar_calibration", "lidar_calibration"))
